vr-wg-charter.html

```diff
@@ -139,8 +139,20 @@ <h1 id="title">[DRAFT] WebVR Working Group Charter</h1>
         <p>With the rising availability of a wide range of head-mounted displays, Virtual Reality (VR) has recently emerged as a critical field of evolution for human-machine interactions.</p>
         <p>The Web provides a promising preexisting ecosystem for the creation, distribution, and experiencing of VR content, applications, and services.</p>
         <p>As explored during the <a href="https://www.w3.org/2016/06/vr-workshop/report.html">October 2016 W3C Workshop</a>, a first step in exploiting that potential relies on allowing browsers to display and react to content adapted to available head-mounted displays. The <a href="https://www.w3.org/community/webvr/">Community-Group incubated</a> <a href="https://w3c.github.io/webvr/">WebVR API </a> has already gained interest from a number of implementors, and this Working Group thus aims at building on that momentum to standardize it as part of the Open Web Platform.</p>
-
-
+        <p>The WebVR Working Group will develop standardized APIs to provide access to input and output capabilities commonly associated with Virtual Reality hardware  such as Google’s Daydream, the Oculus Rift, the Samsung Gear VR, the HTC Vive, and Windows Mixed Reality headsets. In other words, the WG will develop an API to allow Virtual Reality web sites that one can view in a VR headset.  This VR experience must be capable of being embedded in the web of today.</p>
```

Comment:
Suggested rewording:

In other words, the WG will define an API to enable the creation of Virtual Reality web sites that can be viewed in a VR headset. These VR experiences must be embeddable in the Web of today to allow progressive enhancement of existing web sites.